t/cve-2017-3302.t

```diff
@@ -15,10 +9,11 @@ use vars qw($test_dsn $test_user $test_password $test_db);
 use lib 't', '.';
 require "lib.pl";
 eval {
-  $dbh = DBI->connect($test_dsn, $test_user, $test_password, {RaiseError => 1});
+  $dbh = DBI->connect($test_dsn, $test_user, $test_password, {RaiseError => 1, mysql_server_prepare => 1});
+  die 'MySQL version less than 4.1 does not support prepared statements' unless MinimumVersion($dbh, '4.1');
```

Comment:
Yes, because prepared statements depends on both client and server. If code is not compiled, then test file would test something different.